### PR TITLE
Instructions data download own machine 54

### LINKED ISSUE #54: How to get data on my own machine
The https://esmvalgroup.github.io/tutorial/setup.html (after #46 is merged) shows that if I want to use my own machine then I should be able to download CMIP. However it does not mention how to do that. It seems synda is involved, but there is no description what it does or how to install and configure it.

It would be nice to install synda in this section. Something like instructions at https://docs.esmvaltool.org/en/latest/input.html#models

### PR DESCRIPTION
**Pull request by sverhoeven**
# Pull Request checklist

Started adding instructions for #54, this will explain how download url of #100 was found.

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [ ] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [x] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [x] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #54

Also adds CI job to check recipe

### CODE REVIEW

**Comment by JaroCamphuijsen** [comment]
Also see issue #103

File: data/dataset.urls [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Dataset urls required fo tutorial, described at https://esmvalgroup.github.io/ESMValTool_Tutorial/setup.html#using-your-own-machine
+# For Running a recipe episode
+# Variable = thetaoga, model = HadGEM2-ES
```
Comment (JaroCamphuijsen) [inline data/dataset.urls:3]: For the first exercise in the preprocessor episode we also need the variable "thetao" which is much larger (a several GB file for every 10 years) than "thetaoga".
These are not here yet..

Comment (JaroCamphuijsen) [reply data/dataset.urls:3]: But maybe it should be reconsidered whether we need all these different datasets. We could just stick with one or two different ones maybe, something for the next esmvaltool tutorial sprint?

Comment (sverhoeven) [reply data/dataset.urls:3]: Ah I missed thetao because its not present in the solution recipe

Comment (JaroCamphuijsen) [reply data/dataset.urls:3]: Yeah, maybe that's an issue in itself?

Comment (sverhoeven) [reply data/dataset.urls:3]: In #110 once we test solution recipes we would also have caught it.

**Comment by JaroCamphuijsen** [comment]
Seems like the CI data download stalls. Too much data?

**Comment by sverhoeven** [comment]
The data node we are using is down see https://esgf-node.llnl.gov/status/

**Comment by JaroCamphuijsen** [comment]
> The data node we are using is down see https://esgf-node.llnl.gov/status/

That doesn't help...

**Comment by sverhoeven** [comment]
data node is backup and recipe CI job are green

**Comment by sverhoeven** [comment]
Most lint errors have been solved, just long links cannot be solved. Checker can only exclude lines which start with `!` so images.

File: setup.md [hunk 2]
```diff
@@ -253,19 +254,48 @@ local machine and go [here](#gitHub-account-(advanced)).
 If you are planning on running ESMValTool on your own machine, please make sure
 that you are able to download CMIP data and that you have several GB of space
 available to install conda & ESMValTool, but also enough to make a copy of some
-data.
+data (12Gb).
+
+For this tutorial you can download the required data files by
```
Comment (SarahAlidoost) [inline setup.md:259]:
```suggestion
We can download a single data file following the instructions as described below:
```

File: setup.md [hunk 3]
```diff
@@ -253,19 +254,48 @@ local machine and go [here](#gitHub-account-(advanced)).
...
+
+1. Going to the [CMIP5 search page on the DKRZ ESGF node][cmip5-search]
```
Comment (SarahAlidoost) [inline setup.md:261]:
```suggestion
1. Go to the [CMIP5 search page on the DKRZ ESGF node][cmip5-search]
```

File: setup.md [hunk 4]
```diff
@@ -253,19 +254,48 @@ local machine and go [here](#gitHub-account-(advanced)).
...
+2. Performing the following search constraints
```
Comment (SarahAlidoost) [inline setup.md:262]:
```suggestion
2. Perform the following search constraints
```

File: setup.md [hunk 5]
```diff
@@ -253,19 +254,48 @@ local machine and go [here](#gitHub-account-(advanced)).
...
+
+    - Model = HadGEM2-ES
+    - Experiment = historical
+    - Time frequency = mon
+    - Ensemble =  r1i1p1
+    - Variable = thetaoga
+
+3. Press search button
+4. On single single search result press `List files`
```
Comment (SarahAlidoost) [inline setup.md:271]:
```suggestion
4. On single search result press `List files`
```

File: setup.md [hunk 6]
```diff
@@ -253,19 +254,48 @@ local machine and go [here](#gitHub-account-(advanced)).
...
+5. At bottom of page click on `Show All Files`
+6. Find the file which starts with `thetaoga` in the list
+7. Use `HTTP Download` link to download
+   the [thetaoga_Omon_HadGEM2-ES_historical_r1i1p1_185912-200512.nc][theatoga.nc] file
+   to `~/default_inputpath/` directory.
+
+The tutorial needs more data files. All of the data files should be downloaded
```
Comment (SarahAlidoost) [inline setup.md:278]:
```suggestion
However, the tutorial needs more data files. The [dataset.urls][ds] file contains all data set URLs and in comments the used search constraints. All of the data files should be downloaded
```

File: setup.md [hunk 7]
```diff
@@ -253,19 +254,48 @@ local machine and go [here](#gitHub-account-(advanced)).
...
+to `~/default_inputpath/` directory with [wget](https://en.wikipedia.org/wiki/Wget):
```
Comment (SarahAlidoost) [inline setup.md:279]:
```suggestion
to `~/default_inputpath/` directory. 

To download the data, run the following command using [wget](https://en.wikipedia.org/wiki/Wget):
```

File: setup.md [hunk 8]
```diff
@@ -253,19 +254,48 @@ local machine and go [here](#gitHub-account-(advanced)).
...
+However, the tutorial needs more data files. The [dataset.urls][ds] file contains all data set URLs and in comments the used search constraints. All of the data files should be downloaded
+to `~/default_inputpath/` directory with [wget](https://en.wikipedia.org/wiki/Wget):
+
+~~~shell
+wget --no-clobber --input-file \
+  https://github.com/ESMValGroup/ESMValTool_Tutorial/raw/master/data/dataset.urls \
+  --directory-prefix $HOME/default_inputpath/
+~~~
+
+The [dataset.urls][ds] file contains all data set urls and in comments the used search constraints.
```
Comment (SarahAlidoost) [inline setup.md:287]:
```suggestion
```

**Pull request merged**